### PR TITLE
Add grade button

### PR DESCRIPTION
**Pull request by kevinsito**
Involves adding a date picker and validation fields for each new "grade" object. The parsed date object will still be successfully saved in the DB as a datestring.

screen shot 2016-08-08 at 11 37 53 am

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -216,6 +233,14 @@ angular.module("contactsApp", ['ngRoute'])
             $scope.restFormUrl = "";
         }
 
+        $scope.addGrade = function() {
```
Comment (ark369) [inline public/js/app.js:236]: Is this a duplicate of line 195?

File: public/js/app.js [hunk 2]
```diff
@@ -233,4 +258,4 @@ angular.module("contactsApp", ['ngRoute'])
                 $location.path("/restaurant/" + restId);
             }
         }
-    });
+    });
```
Comment (ark369) [inline public/js/app.js]: nit: end files with newlines.

**Comment by ark369** [comment]
LGTM other than 2 comments.

File: public/js/app.js (hunk 1 above)
Comment (kevinsito) [inline public/js/app.js:236]: There are duplicates, but required in both New and Edit controllers.

Comment (ark369) [inline public/js/app.js:236]: Ah, foiled by github's auto-collapsing...

**Pull request merged**